### PR TITLE
Use components and component guide

### PR DESCRIPTION
**Pull request by fofr**
* Use govuk_publishing_components gem to generate guide
* Mount at /component-guide and indicate presence at startup
* Include minutest integration tests for automated accessibility testing

## Components:

* Use pagination component on latest lists
* Use title component on email signup pages
* Use metadata component on subtopics
* Create topic list component

### Before
screen shot 2017-09-26 at 11 21 35

### After
screen shot 2017-09-26 at 11 21 24

### Before
screen shot 2017-09-26 at 11 19 18

### After
screen shot 2017-09-26 at 11 19 02

## Component guide

topic_list

### CODE REVIEW

**Comment by vanitabarrett** [comment]
Have you got links for pages with the pagination and metadata changes ?

**Comment by fofr** [comment]
@vanitabarrett:
https://www.gov.uk/topic/business-tax/vat/latest?start=50
https://www.gov.uk/topic/business-tax/vat
https://www.gov.uk/topic/business-tax/vat/email-signup
https://www.gov.uk/topic/business-tax/
https://www.gov.uk/government/organisations/environment-agency/services-information

File: test/integration/component_guide_test.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'integration_test_helper'
```
Comment (nickcolley) [inline test/integration/component_guide_test.rb]: Should we have a specific test for the new component?

Comment (fofr) [reply test/integration/component_guide_test.rb]: Tests added in 35ef5ef

**Pull request merged**